vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.props" Condition="Exists('..\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.props')" />
```

Comment:
This *gunk* is maintained by visual studio to handle the packages. When UWP upgrades to modern msbuild and supports Packagereference, all this can be removed and we can add the packagereferences in the shared `build.directory.props` file.

Reply:
Note: had to make a small update to the vs generated stuff to handle customers building our code.